### ISSUE TITLE
Is there any plan to support SE mesh, textures? 

### ISSUE THREAD

**Issue by tokitoki22** [niftools/blender_nif_plugin#327]
I know this plug in still developed  and thanks your efforts.
Then at same time,, I feel if there is no plan to support for skyrim SE version nif  and textures?

I know there was request about SE, but after release SE in 2016  I could not see any news about SE
 with blender_nif_plugin.

So what I hope to know is,, blender_nif_plugin developer not plan to support SE meshes,  or 
if there will be another branch,, or plan to release blender_nif_plugin for SE?

**Comment by neomonkeus** [comment]
Its on a very long todo list.
We have added some initial support to our file library to support some of the updates, but there is a design issue where the same block attribute changes type between versions, something the library never foresaw. So that needs to be resolved first.

While that is ongoing, the work we are generally doing is making adding/fixing things so much easier so it a much pronged approach. Main issue is finding time to get everything done.

**Comment by tokitoki22** [comment]
I see, it is good news for me plug in developers have plan for SE, I know it should be long term.
Thanks all yours effort. (nif blender plug-in teams)

**Comment by neomonkeus** [comment]
Yeah,. we have made a lot of progress in getting the plugin to a place that is easier to work with, which will be needed for these type of large scale changes such as adding in new game support & migrating to Blender 2.8.
